helix-gateway/src/main/java/org/apache/helix/gateway/constant/GatewayServiceEventType.java

```diff
@@ -0,0 +1,7 @@
+package org.apache.helix.gateway.constant;
+
+public enum GatewayServiceEventType {
```

Comment:
Why we need this new type? I remember we have one in your last pr.

Reply:
TFTR. I moved the enum to constant folder